components/post_view/date_separator/date_separator.test.jsx

```diff
@@ -22,9 +22,9 @@ describe('components/post_view/DateSeparator', () => {
     test('should render date without timezone enabled', () => {
         const wrapper = mountWithIntl(
             <DateSeparator
-                date={new Date('Fri Jan 12 2018 20:15:13 GMT+0800 (+08)')}
+                date={new Date('Fri Jan 12 2018 20:15:13 GMT+1200 (+12)')}
```

Comment:
I don't understand the logic behind this fix here, is this fixing the problem in your own environment or is fixing the problem for everybody? Why is fixed?

Reply:
It seems its my local environment. I couldn't find a way to pass these two tests without changing them. From what I understand its because I'm in New Zealand and its daylight savings time (so we are an hour forward at present). This was causing the test to return Saturday instead of Friday which is of course a fail. I changed it to that because that is NZ's GMT and we ahead of pretty much every other timezone. I'm happy to change this to something else if there is a better suggestion.

Reply:
Yes, I think that makes sense, thanks for the explanation :)